css-layout-classes/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1>Choose your Character!</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-two-third">
+            <div class="column-one-second"><img src="images/link.png" /></div>
+            <div class="column-one-second">
+              <h2>Super Smash Bros. Ultimate</h2>
+              <p>
+                Super Smash Bros. Ultimate received universal acclaim from
+                both......
+              </p>
+            </div>
+          </div>
+          <div class="column-one-third">
+            <img src="images/yoshi.png" />
+          </div>
```

Comment:
All 3 of these columns should be the same. They should all be a `column-one-third`